.github/workflows/ci_test-tpu.yml

```diff
@@ -1,96 +1,104 @@
 name: TPU tests
 
+# Workflow Steps:
+#  1. Setup gcloud CLI
+#  2. Checkout Pytorch Lightning
+#  3. Set up Python
+#  4. Set up Python 3.7
+#  5. Get Current PR Number
+#  6. Checkout ml-testing-accelerators
+#  7. Install jsonnet
+#  8. Get Cluster Credentials
+#  9. Deploy the job on the kubernetes cluster
+#  10. Statistics
+#  11. Upload coverage results
+#  12. Upload coverage to Codecov
+
 on:
   push:
     branches: [master, "release/*"]
-# TODO: temporal disable TPU testing until we find way how to pass credentials to forked PRs
-#  pull_request:
-#    branches:
-#      - master
 
 env:
   GKE_CLUSTER: lightning-cluster
   GKE_ZONE: us-central1-a
-  IMAGE: gcr.io/${{ secrets.GKE_PROJECT }}/tpu-testing-image
   MAX_CHECKS: 360
   CHECK_SPEEP: 5
 
 jobs:
-  setup-build-publish-deploy:
+  setup-deploy:
     name: tpu-testing-job
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7]
+        python-version: [3.6]
         xla-version: [1.6, 1.7]
     # Timeout: https://stackoverflow.com/a/59076067/4521646
     timeout-minutes: 50
 
     steps:
-    - name: Set IMAGETAG
-      run: echo "IMAGETAG=$(date +%s)_${{ matrix.python-version }}" >> $GITHUB_ENV
+    - name: Setup gcloud CLI
+      uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+      with:
+        version: '290.0.1'
+        service_account_key: ${{ secrets.GKE_SA_KEY_BASE64 }}
+        project_id: ${{ secrets.GKE_PROJECT }}
+        export_default_credentials: true
+
+    - name: Checkout Pytorch Lightning
+      uses: actions/checkout@v2
+      with:
+        repository: PyTorchLightning/pytorch-lightning
+        ref: ${{ github.event.base_ref }}
```

Comment:
why, by default it checkout the current PR/commit

Reply:
I didn't know. Thanks !